src/allmydata/storage/server.py

```diff
@@ -699,6 +690,24 @@ def _allocate_slot_share(self, bucketdir, secrets, sharenum,
                                          self)
         return share
 
+    def list_mutable_shares(self, storage_index) -> set[int]:
```

Comment:
"list" in the name w/ set as the return type gives me a little dissonance.  Maybe "enumerate_" or just "get_" ... or "iter_" and make it return an `Iterable[int]` instead.

Reply:
Also, a type annotation for `storage_index` would be good I guess.